code/game/objects/items/devices/flashlight.dm

```diff
@@ -137,10 +140,11 @@
 	icon_state = "lamp"
 	item_state = "lamp"
 	brightness_on = 5
-	w_class = 4
+	w_class = 5

```

Comment:
It used to be like that. But new Bay storage system fucked some other objects like this as well. IIRC.